source/Mlos.Python/mlos/Optimizers/ExperimentDesigner/UtilityFunctionOptimizers/unit_tests/TestUtilityFunctionOptimizers.py

```diff
@@ -100,7 +100,7 @@ def tearDownClass(cls) -> None:
         temp_dir = os.path.join(os.getcwd(), "temp")

```

Comment:
This comment is not related to this PR, so we can take it for later, but ...
This pattern is sensitive to the current working directory the user happens to be in when they execute the scripts.  I made issue #74 to track it a while back because unit tests were failing if not run from a particular directory (b47737aa3e610c722afe99ef5819d85bc8fd204d).
Would it be better to dump these in a consistent location, regardless of where they were at that point.  For instance, in `out/` where most of the other non-source stuff goes?  Trouble in this case (unlike the unit test case) is it makes examining the debug output more cumbersome because it's not sitting in the directory you're already in.

Reply:
Sure. Let's tackle it as part of #74 though.